Docker_files/ocsci_container/Dockerfile_ocsci

```diff
@@ -24,10 +24,21 @@ ENV OCSCI_INSTALL_DIR=/opt/ocs-ci
 COPY scripts/install-ocs-ci.sh /usr/local/bin/
 RUN chmod 755 /usr/local/bin/*.sh
 
+# Copy edit_yaml.py script inside the container.
+COPY scripts/edit_yaml.py /opt
```

Comment:
Simplify with `COPY --chmod 755`

Reply:
I got this error for docker
 ```Step 5/17 : COPY --chmod=755 scripts/install-ocs-ci.sh /usr/local/bin/
the --chmod option requires BuildKit. Refer to https://docs.docker.com/go/buildkit/ to learn how to build images with BuildKit enabled
make: *** [Makefile:11: build-image] Error 1
```

for podman is working as expected